tasks/configure.yml

```diff
@@ -25,3 +25,24 @@
     name: datadog-agent
     state: started
   when: datadog.autostart_agent
+
+- name: Change Datadog Upstart Runlevel for rc0.d and rc6.d
+  become: yes
+  file:
+    src: "/etc/init.d/datadog-agent"
+    dest: "/etc/rc0.d/K05datadog-agent"
+    owner: root
+    group: root
+    state: link
+  file:
+    src: "/etc/init.d/datadog-agent"
+    dest: "/etc/rc6.d/K05datadog-agent"
+    owner: root
+    group: root
+    state: link
+  file:
+    path: /etc/rc0.d/K20datadog-agent
+    state: absent
+  file:
+    path: /etc/rc6.d/K20datadog-agent
+    state: absent
```

Comment:
I don't like this being chained like this, can we separate into two tasks, one for the links and one for the deletes, with variables used to provide the paths?